eido/conversion.py

```diff
@@ -53,6 +53,8 @@ def run_filter(prj, filter_name, plugin_kwargs=None):
     """
     installed_plugins = pep_conversion_plugins()
     installed_plugin_names = list(installed_plugins.keys())
+    path = plugin_kwargs.get("path")
```

Comment:
looks like plugin_kwargs isn't defined until line 65, is that right?

Reply:
Do you think this parameter should be `paths` instead of `path` if it is intended to be a dict with one or more paths?

Reply:
That might make more sense, yes.

Reply:
`path` is used, however, extensively throughout the package and docs... I wouldn't want to break any backward compatibility if we switched to `paths`.

Reply:
yes, but I've just realized that as coded, it's in fact just a `path`, and not a `paths`.

I think this needs to be changed to be `paths`. How it's set up now isn't what I thought we had concluded.